Library/Homebrew/prefix.sh

```diff
@@ -17,10 +17,10 @@ homebrew-prefix() {
   [ -z "$formula" ] && echo "$HOMEBREW_PREFIX" && return 0
 
   local formula_path
-  if [ -f "$HOMEBREW_REPOSITORY/Library/Taps/homebrew/homebrew-core/${formula}.rb" ]; then
-    formula_path="$HOMEBREW_REPOSITORY/Library/Taps/homebrew/homebrew-core/${formula}.rb"
+  if [ -f "$HOMEBREW_REPOSITORY/Library/Taps/homebrew/homebrew-core/Formula/${formula}.rb" ]; then
+    formula_path="$HOMEBREW_REPOSITORY/Library/Taps/homebrew/homebrew-core/Formula/${formula}.rb"
   else
-    formula_path="$(find "$HOMEBREW_REPOSITORY/Library/Taps" -name "${formula}.rb" -print -quit)"
+    formula_path="$(find "$HOMEBREW_REPOSITORY/Library/Taps" -path "*/Formula/${formula}.rb" -print -quit)"
```

Comment:
This will incorrectly filter out results for taps where formulae are in the tap root.

Reply:
Ah, forgot about the following paragraph in https://docs.brew.sh/How-to-Create-and-Maintain-a-Tap:
> Tap formulae follow the same format as the core’s ones, and can be added under either the `Formula` subdirectory, the `HomebrewFormula` subdirectory or the repository’s root.

Will fix.